src/components/Header/index.js

```diff
@@ -123,6 +136,14 @@ const PublicMenuOption = () => {
   );
 };
 
+const PublicLeftMenuOption = () => {
+  return (
+    <a className="item" href={PATHS.COURSE}>
+      Courses
+    </a>
+  );
+};
```

Comment:
I think it might be good to create an object that maps the path keys to human-readable names.  E.g.,
```
const NAMES = {
  COURSE: "Courses",
  MENTOR: "Mentors",
  CLASS: "Classes"
};
```
You can add the names for the other links as well.  The point is that there could be an object constant like this for each supported language (e.g., Hindi, Tamil, etc.) so if the user selects a different language, it would be a simple matter of reassigning `NAMES`.  

Then I'd also recommend creating a component that contains a single navigation menu option like this:
```
const MenuOption = (props) => {
  return (
    <a className="item" href={PATHS[props.id]}>
      {NAMES[props.id]}
    </a>
  );
};
```
So for example, `AuthenticatedLeftHeaderOption` would look like this 
```
const AuthenticatedLeftHeaderOption = () => {
  return (
    <>
      {["COURSE", "MENTOR", "CLASS"].map((id) => (
        <MenuOption id={id} />
      ))}
    </>
  );
};
```
This would make it easy to say change an `a` to `Link`.

Reply:
But in `AuthenticatedHeaderOption` we are using another className. There is 2 classNames `item` and `left-item`, how about that? if you're asking to do the same with this `AuthenticatedHeaderOption` function as well.